test cases/frameworks/7 gnome/resources-data/meson.build

```diff
@@ -0,0 +1,14 @@
+subdir('subdir')
+
+script = '''
+import os, sys
+assert os.path.exists(sys.argv[1]), "File %s not found" % sys.argv[1]
+print("Unimportant generated content")
+'''
+
+res3_txt = custom_target('res3.txt',
+  input: 'res3.txt.in',
+  output: 'res3.txt',
+  command: ['python', '-c', script, '@INPUT@'],
+  capture: true,
+)
```

Comment:
I thought that it did generate output... I've tested though and you are correct. Could you explain why 'capture' doesn't behave as documented at https://github.com/mesonbuild/meson/wiki/Reference%20manual#custom_target ?

> When this argument is set to true, Meson captures stdout and writes it to the target file.

That suggests to me that this snippet should generate a file called res3.txt that contains the string "Unimportant generated content", which is what I intended it to do :-)

Of course the script could equally take 2 arguments and write to the file itself, I'm happy to change the code to work this way if that's preferred.. but I'm missing something about why it doesn't work as-is